spec/features/card_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'Reviewing card' do
+  subject! { create(:card) }
+
+  context 'When there are cards for review' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.